imsim/stamp.py

```diff
@@ -480,6 +480,11 @@ def fix_seds(prof):
             # can blow up the required FFT size.  We'll guard for that below with
             # a try block, but we can minimize how often this happens by making sure
             # the offset is close to 0,0.
```

Comment:
This comment is now in the wrong place.  It refers to the below test re offset.  But I think we agreed a while back that that block is broken and we should just remove it.  You can probably go ahead and do that on this PR.